webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -67,14 +66,13 @@ export const combineMenuItems = data => {
 
 const createOrgItem = orgs => {
   const anyOrg = {
-    name: ANY_ORGANIZATION_TEXT,
+    name: __('Any Organization'),
     url: '/organizations/clear',
```

Comment:
Unrelated but this should also use the URL helper.

Reply:
this will probably change with https://github.com/theforeman/foreman/pull/8243/ so i'll leave it to @MariaAga